### PR TITLE
CIE-4459. Updated the note.

### PR DESCRIPTION
**Pull request by sharath-jk**

### CODE REVIEW

File: content/edge/rest-api.md [hunk 1]
```diff
@@ -5,3 +5,12 @@ layout: bundle
 ---
 
 This section describes the REST APIs for {{< product-c8y-iot >}} Edge.
+
+{{< c8y-admon-important >}}
+
+Wherever possible, the REST resources and its representations are maintained in a *backwards compatible* manner. However, the Edge installation or configuration scripts developed based on the REST API of an older Edge version (for example, 10.9.0.0) may not work against the newer version (for example, 10.15.0.0). Also, the behavior of the REST API may change without warning if the existing behavior is incorrect or constitutes a security vulnerability.
+
+This API is not covered by the general Cumulocity IoT API Backwards compatibility statement mentioned here: [https://cumulocity.com/guides/concepts/introduction/#compatibility-policy](https://cumulocity.com/guides/concepts/introduction/#compatibility-policy)
```
Comment (BeateRixen) [inline content/edge/rest-api.md:13]:
```suggestion
This API is not covered by the general Cumulocity IoT API backwards compatibility statement mentioned here: [https://cumulocity.com/guides/concepts/introduction/#compatibility-policy](https://cumulocity.com/guides/concepts/introduction/#compatibility-policy)
```

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@sharath-jk This PR needs another approval by your team.

File: content/edge/rest-api.md [hunk 2]
```diff
@@ -5,3 +5,12 @@ layout: bundle
 ---
 
 This section describes the REST APIs for {{< product-c8y-iot >}} Edge.
+
+{{< c8y-admon-important >}}
+
+Wherever possible, the REST resources and its representations are maintained in a *backwards compatible* manner. However, the Edge installation or configuration scripts developed based on the REST API of an older Edge version (for example, 10.9.0.0) may not work against the newer version (for example, 10.15.0.0). Also, the behavior of the REST API may change without warning if the existing behavior is incorrect or constitutes a security vulnerability.
```
Comment (bbyreddy) [inline content/edge/rest-api.md:11]: Is this _"Also, the behavior of the REST API may change without warning if the existing behavior is incorrect or constitutes a security vulnerability."_ statement mentioning _**"without warning"**_  fine? If not, please change it to _**without advanced notice**_ to keep it a bit subtle.

**Comment by sharath-jk** [comment]
> Minor change, but in general, I am fine with it. However, doesn't core API come with such a warning? If so, we should align this text with that of the core.

@bbyreddy - a similar text is mentioned in the **Introduction** section here **https://cumulocity.com/api/**. This page is common for all Cumulocity IoT modules.

File: content/edge/rest-api.md (hunk 2 above)
Comment (sharath-jk) [reply content/edge/rest-api.md]: Updated the text.

**Review by bbyreddy — APPROVED** [review]

**Pull request merged**